l10n_eu_nace/tests/test_en_nace_request_results.py

```diff
@@ -137,7 +137,7 @@ def create_response(*lists):
     response = Response()
     response.code = "200"
     response.status_code = 200
-    merged_lists = list(zip(*lists))
+    merged_lists = list(zip(*lists, strict=False))
```

Comment:
The `strict=False ` would not be necessary. By default it is already in the library.
https://peps.python.org/pep-0618/#add-itertools-zip-strict

Reply:
Giving a value to the "strict" parameter is required by pre-commit.

Reply:
aps... @pedrobaeza @etobella do u know why is required in pre-commit? 😄

Reply:
No idea, but it's not a big deal IMO

Reply:
It's more curiosity haha thanks :)